manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: 70.30.5981
+PackageLocale: en-US
+Publisher: Araxis Ltd.
```

```suggestion
Publisher: Araxis
```

Should match Control Panel / Apps & Features for better package correlation.

Or you can retain `Araxis Ltd.` here to show this with `winget show` command but add `Publisher: Araxis` under Apps&FeaturesEntries in the installer yaml for providing exact match with Registry entry.